docs/DeveloperGuide.md

```diff
@@ -975,19 +916,95 @@ testers are expected to do more *exploratory* testing.
 In the interest of transparency and keeping our users informed, we have identified some aspects
 of our product that we recognise as feature flaws. We appreciate your patience as
 we are actively working to find the best solution to address these feature flaws.
 
+### Editing a pair by index: `edit_pair`
+
+Currently, editing a pair by index has not been implemented. This feature can increase the efficiency of using FriendlyLink.
+This feature was originally deemed low priority, since a pair only has 2 fields: elderly and volunteer, and thus users can reasonably still achieve this by deleting a pair and adding a new pair. Through user feedback, we acknowledge that this is still a useful feature to have to improve efficiency and user experience.
+
+Proposed usage: Edits an existing pair based on their index in the pairs list.
+Proposed format: `edit_pair INDEX [eic/ELDERLY_NRIC] [vic/VOLUNTEER_NRIC]`
+
+Proposed behaviour:
+* Edits the pair at the specified `INDEX` in the displayed pair list.
+* Any combination of the optional fields is possible but **at least one** optional field must be specified.
+* Existing values will be updated to the input values.
+
+Examples:
+* `edit_pair 1 eic/T0245267I` Edits the 1st pair so that the volunteer is paired to the elderly with NRIC `T0245267I` instead.
+
+### Deleting a pair by index: `delete_pair`
+
+Currently, pairs are deleted by specifying NRIC of both elderly and volunteer. This feature can increase the efficiency of using FriendlyLink, if we allow users to simply specify the index of the pair.
+This feature is originally implemented as such to prevent accidental deletion of pairs, as it is easy to enter the wrong index but hard to accidentally enter a pair of incorrect NRICs and delete the wrong pair. Through user feedback, we acknowledge that we should implement it to delete by index and support this with an `undo` feature to minimise impact of accidental deletions.
+
+Proposed usage: Deletes an existing pair based on their index in the pairs list.
+Proposed format: `edit_pair INDEX [eic/ELDERLY_NRIC] [vic/VOLUNTEER_NRIC]`
+
+Proposed behaviour:
+* Edits the pair at the specified `INDEX` in the displayed pair list.
+* Any combination of the optional fields is possible but **at least one** optional field must be specified.
+* Existing values will be updated to the input values.
+
+### Display birthdate in person cards
+
+When users provide a birthdate for an Elderly or Volunteer profile, only the person's current age is displayed in the app. To view the specific birthdate, users must refer to the respective JSON file.
+We should display the birthdate in FriendlyLink as well.
+
+### Enhance display of large number of tags
+
+There are no limits on the number of tags or the character limit for each tag. If large number of tags or tags with excessively long names are used, it affects the UI display and impact user experience.
+<Insert proposed solution>
```

Comment:
Currently, there are no limits on the number of tags or the character limit for each tag. If large number of tags or tags with excessively long names are used, it affects the UI display and impact user experience.

Therefore, in v 1.5, we plan to limit each tag within 50 characters, with the total number of tags being less than 10.
This would ensure decent degree of input flexibility, while not severely clogging up the UI display.